go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/cosmos/gogoproto v1.4.11
 	github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v7 v7.1.1
 	github.com/cosmos/ibc-apps/modules/async-icq/v7 v7.1.1
+	github.com/cosmos/ibc-go/modules/light-clients/08-wasm v0.0.0-20231213092633-b306e7a706e1
```

Comment:
What's the reason for using this commit instead of a tagged version?

Reply:
ah good catch, I was trying out bunch of commits and forgot to change this.